app/uk/gov/hmrc/helptosavestub/controllers/CitizenDetailsController.scala

```diff
@@ -52,8 +52,9 @@ object CitizenDetailsController extends BaseController {
     val personGen = for {
       fnameO <- Gen.forename.almostAlways
       snameO <- Gen.surname.almostAlways
```

Comment:
As discussed, almostAlways can return null/blank - for what we're doing at the moment, I think we need a forename and surname consistently returned.